aries-site/src/examples/components/select/SelectExample.js

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+import { Box, Select } from 'grommet';
+
+const options = ['Item One', 'Item Two', 'Item Three', 'Item Four'];
```

Comment:
Seems like designs call for 18px font-size for options but this has 16px

Reply:
Dependent on the Theme PR https://github.com/grommet/grommet-theme-hpe/pull/46